aikido/src/util/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ aikido_add_library("${PROJECT_NAME}_util"
   VanDerCorput.cpp
   StepSequence.cpp
   RNG.cpp
+  stream.cpp
```

Comment:
I thought naming it `Stream.hpp` would imply that the file contains a class named `Stream`. However, we already break this rule in `PseudoInverse.hpp`, so that ship has already sailed. 😁